documentation/modules/overview/con-security-configuration-fips.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// overview/assembly-security-overview.adoc
+
+[id="security-configuration-fips_{context}"]
+= Federal Information Processing Standards (FIPS)
+
+[role="_abstract"]
+Federal Information Processing Standards (FIPS) are a set of security standards established by the US government to ensure the confidentiality, integrity, and availability of sensitive data and information that is processed or transmitted by information systems. 
+Strimzi supports FIPS compliance by automatically enabling FIPS mode on the OpenJDK used in Strimzi container images when running on a FIPS-enabled Kubernetes cluster.
+
+FIPS compliance requires the use of specific cryptographic algorithms and security protocols, which are supported in a FIPS-enabled Kubernetes cluster.
+Since switching to FIPS mode on an existing Kubernetes cluster or migrating to a FIPS-enabled cluster can be complex, it's easier to deploy Strimzi on a FIPS-enabled cluster from the outset.
```

Comment:
I don't think we really know that. I also don't think Strimzi cares about this. So I would leave it out.

Reply:
removed